README.md

```diff
@@ -835,7 +835,8 @@ const results = window.markdownlint.sync(options).toString();
 
 ## Examples
 
-For ideas how to integrate `markdownlint` into your workflow, refer to the following projects:
+For ideas how to integrate `markdownlint` into your workflow, refer to the
+[Related section](#Related) or the following projects:
```

Comment:
How about: "refer to the following projects or one of the tools in the [Related section]:"